diff --git a/pkg/test/echo/docker/Dockerfile.app b/pkg/test/echo/docker/Dockerfile.app
index 19b2a9554d..de49cc02b2 100644
--- a/pkg/test/echo/docker/Dockerfile.app
+++ b/pkg/test/echo/docker/Dockerfile.app
@@ -1,13 +1,15 @@
-ARG BASE_VERSION=latest
-ARG ISTIO_BASE_REGISTRY=gcr.io/istio-release
+#ARG BASE_VERSION=latest
+#ARG ISTIO_BASE_REGISTRY=gcr.io/istio-release

-FROM ${ISTIO_BASE_REGISTRY}/base:${BASE_VERSION}
+#FROM ${ISTIO_BASE_REGISTRY}/base:${BASE_VERSION}
+
+FROM ubuntu:24.04

 ARG TARGETARCH
-COPY ${TARGETARCH:-amd64}/client /usr/local/bin/client
-COPY ${TARGETARCH:-amd64}/server /usr/local/bin/server
-COPY certs/cert.crt /cert.crt
-COPY certs/cert.key /cert.key
+COPY client /usr/local/bin/client
+COPY server /usr/local/bin/server
+COPY cert.crt /cert.crt
+COPY cert.key /cert.key

 # Add a user that will run the application. This allows running as this user and capture iptables
 RUN useradd -m --uid 1338 application
