diff --git a/samples/helloworld/helloworld.yaml b/samples/helloworld/helloworld.yaml
index 68e1072..52a61b6 100644
--- a/samples/helloworld/helloworld.yaml
+++ b/samples/helloworld/helloworld.yaml
@@ -33,7 +33,7 @@ spec:
     spec:
       containers:
       - name: helloworld
-        image: quay.io/sail-dev/examples-helloworld-v1:1.0
+        image: quay.io/maistra/helloworld-v1:1.0
         resources:
           requests:
             cpu: "100m"
@@ -62,7 +62,7 @@ spec:
     spec:
       containers:
       - name: helloworld
-        image: quay.io/sail-dev/examples-helloworld-v2:1.0
+        image: quay.io/maistra/helloworld-v2:1.0
         resources:
           requests:
             cpu: "100m"
diff --git a/samples/httpbin/httpbin.yaml b/samples/httpbin/httpbin.yaml
index 5f115ad..736476b 100644
--- a/samples/httpbin/httpbin.yaml
+++ b/samples/httpbin/httpbin.yaml
@@ -53,7 +53,7 @@ spec:
     spec:
       serviceAccountName: httpbin
       containers:
-      - image: quay.io/sail-dev/examples-httpbin:v2.15.0
+      - image: quay.io/openshifttest/httpbin:1.2.2
         imagePullPolicy: IfNotPresent
         name: httpbin
         ports:
diff --git a/samples/sleep/sleep.yaml b/samples/sleep/sleep.yaml
index eeb4a91..6e06848 100644
--- a/samples/sleep/sleep.yaml
+++ b/samples/sleep/sleep.yaml
@@ -52,7 +52,7 @@ spec:
       serviceAccountName: sleep
       containers:
       - name: sleep
-        image: quay.io/curl/curl
+        image: quay.io/curl/curl:8.4.0
         command: ["/bin/sleep", "infinity"]
         imagePullPolicy: IfNotPresent
         volumeMounts:
diff --git a/samples/tcp-echo/tcp-echo-ipv4.yaml b/samples/tcp-echo/tcp-echo-ipv4.yaml
index 7ecbda7..55f19eb 100644
--- a/samples/tcp-echo/tcp-echo-ipv4.yaml
+++ b/samples/tcp-echo/tcp-echo-ipv4.yaml
@@ -53,7 +53,7 @@ spec:
     spec:
       containers:
       - name: tcp-echo
-        image: quay.io/sail-dev/examples-tcp-echo-server:1.3
+        image: quay.io/maistra/-tcp-echo-server:1.3
         imagePullPolicy: IfNotPresent
         args: [ "9000,9001,9002", "hello" ]
         ports:
diff --git a/tests/e2e/common-operator-integ-suite.sh b/tests/e2e/common-operator-integ-suite.sh
index bb225f2..8af56cb 100755
--- a/tests/e2e/common-operator-integ-suite.sh
+++ b/tests/e2e/common-operator-integ-suite.sh
@@ -289,4 +289,4 @@ fi
 IMAGE="${HUB}/${IMAGE_BASE}:${TAG}" SKIP_DEPLOY="${SKIP_DEPLOY}" OCP="${OCP}" IP_FAMILY="${IP_FAMILY}" ISTIO_MANIFEST="${ISTIO_MANIFEST}" \
 NAMESPACE="${NAMESPACE}" CONTROL_PLANE_NS="${CONTROL_PLANE_NS}" DEPLOYMENT_NAME="${DEPLOYMENT_NAME}" MULTICLUSTER="${MULTICLUSTER}" ARTIFACTS="${ARTIFACTS}" \
 ISTIO_NAME="${ISTIO_NAME}" COMMAND="${COMMAND}" KUBECONFIG="${KUBECONFIG}" ISTIOCTL_PATH="${ISTIOCTL}" \
-go run github.com/onsi/ginkgo/v2/ginkgo -tags e2e --timeout 60m --junit-report=report.xml ${GINKGO_FLAGS} "${WD}"/...
+go run github.com/onsi/ginkgo/v2/ginkgo -tags e2e --timeout 60m --junit-report=report.xml --skip "Ambient" ${GINKGO_FLAGS} "${WD}"/...
diff --git a/tests/e2e/controlplane/control_plane_suite_test.go b/tests/e2e/controlplane/control_plane_suite_test.go
index 10317f7..23a84fe 100644
--- a/tests/e2e/controlplane/control_plane_suite_test.go
+++ b/tests/e2e/controlplane/control_plane_suite_test.go
@@ -43,7 +43,7 @@ var (
 	istioCniNamespace     = env.Get("ISTIOCNI_NAMESPACE", "istio-cni")
 	istioCniName          = env.Get("ISTIOCNI_NAME", "default")
 	skipDeploy            = env.GetBool("SKIP_DEPLOY", false)
-	expectedRegistry      = env.Get("EXPECTED_REGISTRY", "^docker\\.io|^gcr\\.io")
+	expectedRegistry      = env.Get("EXPECTED_REGISTRY", "^docker\\.io|^gcr\\.io|^registry\\.redhat\\.io")
 	sampleNamespace       = env.Get("SAMPLE_NAMESPACE", "sample")
 	multicluster          = env.GetBool("MULTICLUSTER", false)
 	ipFamily              = env.Get("IP_FAMILY", "ipv4")
diff --git a/tests/e2e/operator/operator_install_test.go b/tests/e2e/operator/operator_install_test.go
index d3267d6..521a988 100644
--- a/tests/e2e/operator/operator_install_test.go
+++ b/tests/e2e/operator/operator_install_test.go
@@ -161,7 +161,7 @@ spec:
     spec:
       containers:
       - name: curl-metrics
-        image: quay.io/curl/curl:8.11.1
+        image: quay.io/curl/curl:8.4.0
         command: ['curl', '-v', '-k', '-H', 'Authorization: Bearer %s', 'https://%s.%s.svc.cluster.local:8443/metrics']
       restartPolicy: Never
 `, curlNamespace, token, metricsServiceName, namespace))
